stablehlo/CMakeLists.txt

```diff
@@ -19,3 +19,6 @@ add_subdirectory(tests)
 add_subdirectory(testdata)
 add_subdirectory(tools)
 add_subdirectory(transforms)
+
+# Optional conversion from StableHLO to other community dialects.
+add_subdirectory(conversions)
```

Comment:
This is no longer optional, right? Let's remove the comment and order alphabetically with respect to other `add_subdirectory` items?